src/wp-admin/includes/class-wp-privacy-policy-content.php

```diff
@@ -450,7 +450,7 @@ public static function privacy_policy_guide() {
 	 * @param bool $blocks      Whether to format the content for the block editor. Default true.
 	 * @return string The default policy content.
 	 */
-	public static function get_default_content( $description = false, $blocks = true ) {
+	public static function get_default_content( $description = false, $blocks = false ) {
```

Comment:
Do we need to keep `$blocks` parameter at all?

Reply:
I think it makes some sense to leave it in there for 2 reasons:
1/ the function is called in a couple of places passing 2 parameters, so if we remove the `$blocks` parameter we'd need to update those calls too.
2/ leaving the second parameter may be useful if the function ever gets called by a plugin or theme, it is declared as `public`.